Apps/frontend/src/views/Oscilloscope.svelte

```diff
@@ -93,7 +93,7 @@
         />
       </div>
       <div class="slider-wrapper">
-        <div class="sliders">
+        <div class="switches">
           Start/Stop
           <br>
           {#each { length: NUM_CHANNELS } as _, index}
```

Comment:
When merging, please make sure to keep updating the indicators' start/stop state in the on:startStop function.